packages/slate-react/src/utils/clone-fragment.js

```diff
@@ -88,7 +89,7 @@ function cloneFragment(event, value, fragment = value.fragment) {
 
   // For browsers supporting it, we set the clipboard registers manually,
   // since the result is more predictable.
-  if (event.clipboardData && event.clipboardData.setData) {
+  if (event.clipboardData && event.clipboardData.setData && !IS_IE) {
```

Comment:
Does this mean IE has those methods, but they don't work?

Reply:
It supports the method, but in a very restricted sense. Specifically, IE doesn't support arbitrary MIME types or even common ones like 'text/plain'; it only accepts the strings "Text" (which gets mapped to 'text/plain') and "Url" (mapped to 'text/url-list') as types for calls to `DataTransfer.setData()`. That's why I suggest we skip the following block if we are in IE.

Reply:
Sounds like this one can't be polyfilled, so I'd be down to accept it alone. Although can you add a `COMPAT:` comment (check other parts of codebase to see one) to explain why. Thanks!